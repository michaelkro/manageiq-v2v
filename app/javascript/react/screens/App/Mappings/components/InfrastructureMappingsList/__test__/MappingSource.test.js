import React from 'react';
import { shallow } from 'enzyme';

import MappingSource from '../components/MappingSource';

test('renders MappingSource with children', () => {
  const wrapper = shallow(
    <MappingSource>
      <div>hello</div>
    </MappingSource>
  );

  expect(wrapper).toMatchSnapshot();
});
