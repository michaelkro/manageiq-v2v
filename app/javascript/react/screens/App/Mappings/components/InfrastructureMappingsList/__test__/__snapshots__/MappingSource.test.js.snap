// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MappingSource with children 1`] = `
<div
  className="infra-mapping-source-container"
>
  <div
    className="infra-mapping-item-source"
  >
    <div>
      hello
    </div>
  </div>
  <div
    className="arrow-wrapper"
  >
    <div
      className="arrow-stem"
    />
    <div
      className="arrow-head"
    />
  </div>
</div>
`;
